src/calculator.js

```diff
@@ -1,48 +1,31 @@
+module.exports = exports;
+var x;
+var y;
```

Comment:
All &#39;var&#39; declarations must be at the top of the function scope.

###### See http://eslint.org/docs/rules/vars-on-top for details about this rule